### PR TITLE
Update to version 4.0.0

### PR DESCRIPTION
**Pull request by petebunting**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by petebunting** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-  url: https://github.com/remotesensinginfo/arcsi/archive/v{{ version }}.tar.gz
-  sha256: 83b069487e363b26de6a2708256b55846e19a6a8a906086ac025152268208877
+  url: https://github.com/remotesensinginfo/arcsi/archive/refs/tags/{{ version }}.tar.gz
+  sha256: ff78ccbb4622c4ec59b66f0309d5716ba2c70b0b13b8df53e75909f27380ee2b
 
 build:
-  number: 2
-  skip: True  # [win and py27]
+  number: 0
+  skip: True  # [win]
```
Comment (mbargull) [inline recipe/meta.yaml:13]: Was this change intentional?
Previously this only skipped Python 2.7 on Windows (since we don't build for Python 2.7 for quite a while, this was essentially ignored).
Now no Windows builds are created at all. To re-enable Windows builds you could issue a PR with simply:
```suggestion
```
